docker/apm-server/managed/docker-entrypoint.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -e
+
+echo ${KIBANA_HOST}
+
+fleet="${KIBANA_HOST}/api/fleet"
+pkg_policy_name="apm-integration-testing"
+
+# remove existing apm-integration-testing package policy
+pkg_policy_id="$(curl -s -H 'kbn-xsrf: true' -H 'Content-Type: application/json' "${fleet}/package_policies?kuery=ingest-package-policies.name:${pkg_policy_name}" | jq '.items[0].id')"
```

Comment:
Why is deletion necessary, is it created other than by this entrypoint?  Can we instead only create if it doesn't already exist?

Reply:
switched to Go implementation, but still deleting any APM packages - the configuration options might not be the same (currently only the secret token).